cmd/svcat/svcat_test.go

```diff
@@ -84,6 +84,7 @@ func TestCommandOutput(t *testing.T) {
 		{name: "list all plans", cmd: "get plans", golden: "output/get-plans.txt"},
 		{name: "get plan by name", cmd: "get plan default", golden: "output/get-plan.txt"},
 		{name: "get plan by uuid", cmd: "get plan --uuid 86064792-7ea2-467b-af93-ac9694d96d52", golden: "output/get-plan.txt"},
+		{name: "get plan by class/plan name combo", cmd: "get plan user-provided-service/default", golden: "output/get-plan.txt"},
```

Comment:
Let's also add a testcase for `describe plan user-provide-service/default` just to test the wiring.